R/froll.R

```diff
@@ -91,12 +91,14 @@ frollapply = function(x, n, FUN, ..., fill=NA, align=c("right", "left", "center"
     n = partial2adaptive(x, n, align)
     adaptive = TRUE
   }
+  if (isTRUE(adaptive) && base::getRversion() < "3.4.0") ## support SET_GROWABLE_BIT
+    stopf("frollapply adaptive=TRUE requires at least R 3.4.0"); # nocov
```

Comment:
let's just bump to 3.4.0 dependency for 1.18.0.

Reply:
this will be rewritten in https://github.com/Rdatatable/data.table/pull/5575 and then it will also work on < 3.4.0 but with an extra copy on each iteration.